src/components/Avatar/__tests__/Avatar.test.js

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { cleanup, render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Avatar from '../Avatar';
+
+afterEach(cleanup); // eslint-disable-line
+
+describe('Avatar', () => {
+  it('should render component with default props', () => {
+    const tree = renderer.create(<Avatar />).toJSON();
+    expect(tree).toMatchInlineSnapshot(`
+      <div
+        className="str-chat__avatar str-chat__avatar--circle"
+        data-testid="avatar"
+        onClick={[Function]}
+        onMouseOver={[Function]}
+        style={
+          Object {
+            "flexBasis": 32,
+            "fontSize": 16,
+            "height": 32,
+            "lineHeight": "32px",
+            "width": 32,
+          }
+        }
+      >
+        <div
+          className="str-chat__avatar-fallback"
+          data-testid="avatar-fallback"
+        />
+      </div>
+    `);
+  });
+
+  it('should render component with default props and image prop', () => {
+    const tree = renderer.create(<Avatar image="random" />).toJSON();
+    expect(tree).toMatchInlineSnapshot(`
+      <div
+        className="str-chat__avatar str-chat__avatar--circle"
+        data-testid="avatar"
+        onClick={[Function]}
+        onMouseOver={[Function]}
+        style={
+          Object {
+            "flexBasis": 32,
+            "fontSize": 16,
+            "height": 32,
+            "lineHeight": "32px",
+            "width": 32,
+          }
+        }
+      >
+        <img
+          alt={null}
```

Comment:
@jaapbakker88 @vishalnarkhede Class component was rendering null value for alt text, should we update this to empty string?

Reply:
yep